pkg/perf/images/create_images.go

```diff
@@ -0,0 +1,199 @@
+package images
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	godigest "github.com/opencontainers/go-digest"
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+func CreateImages(parallelImagesNumber int) error {
+	pushedImages := GetImagesName()
+	for i := 0; i < len(pushedImages); i++ {
+		imageDir := fmt.Sprintf(imageDirPath, pushedImages[i])
+		if _, err := os.Stat(imageDir); os.IsNotExist(err) {
+			if err := MakeImageDir(imageDir); err != nil {
+				return err
+			}
+
+			if err := CreateLayout(imageDir); err != nil {
+				log.Fatalln(err)
+			}
+
+			manifestName, manifestSize := CreateBlobs(size, imageDir)
```

Comment:
just need this, also "imageDir" must be a variable, so we can pick the fastest storage or even in some cases ramfs.